lib/connectionConfig/eventhubConnectionConfig.ts

```diff
@@ -80,7 +80,7 @@ export module EventHubConnectionConfig {
   export function create(connectionString: string, path?: string): EventHubConnectionConfig {
     const config = ConnectionConfig.create(connectionString, path);
     if (!config.entityPath) {
-      throw new Error(`Either provide "path" or the "connectionString": "${connectionString}", ` +
+      throw new TypeError(`Either provide "path" or the "connectionString": "${connectionString}", ` +
```

Comment:
I think we can skip this check. `validate()` would catch the case of missing entityPath for Event Hubs right?

Reply:
We are not calling `validate()` in Event Hubs anywhere.

Reply:
Ok